src/components/grid_overlay/grid_overlay.ts

```diff
@@ -38,16 +38,21 @@ function useCellHovered(
   const { pause, resume } = useInterval(checkTiming, 200);
 
   function checkTiming() {
-    const { col, row } = getPosition();
-    const delta = Date.now() - lastMoved;
-    if (delta > 300 && (col !== hoveredPosition.col || row !== hoveredPosition.row)) {
-      setPosition(undefined, undefined);
-    }
-    if (delta > 300) {
-      if (col < 0 || row < 0) {
-        return;
+    try {
```

Comment:
the try/catch should be moved to `useInterval` IMO

Reply:
Was thinking what if we want to have the useInterval callback to handle thrown errors ? But I was stupid, we can still do a try/catch in the callback 🙈